src/webgpu/api/validation/capability_checks/limits/maxStorageBufferBindingSize.spec.ts

```diff
@@ -146,7 +152,12 @@ g.test('validate,maxBufferSize')
     const { defaultLimit, maximumLimit } = t;
     const requestedLimit = getDeviceLimitToRequest(limitTest, defaultLimit, maximumLimit);
 
-    await t.testDeviceWithSpecificLimits(requestedLimit, 0, ({ device, actualLimit }) => {
-      t.expect(actualLimit <= device.limits.maxBufferSize);
-    });
+    await t.testDeviceWithSpecificLimits(
```

Comment:
This test can be simplified to just this

```
g.test('validate,maxBufferSize')
  .desc(`Test that ${limit} <= maxBufferSize`)
  .fn(t => {
    const { limitTest } = t.params;
    const { adapter, defaultLimit, maximumLimit } = t;
    t.expect(defaultLimit <= t.getDefaultLimit('maxBufferSize'))
    t.expect(maximumLimit <= adapter.limits.maxBufferSize);
  });
```

Sorry, it was on my to-do list to cleanup as this test pre-exists some other changes in `limit_utils.ts`

same in the other file.

Reply:
Done!